URBNSwiftCarousel/Classes/URBNCarouselZoomableCellCollectionViewCell.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  URBNCarouselZoomableCellCollectionViewCell.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 4/26/16.
+//
+//
+
+import UIKit
+
+/**
+ 
+ This subclass of UICollectionViewCell has a zoomable UIImageView on top of a scrollView.
+ 
+*/
+public class URBNCarouselZoomableCell: UICollectionViewCell, UIScrollViewDelegate {
```

Comment:
TODO - apply a protocol to the collection view cell that extracts a uiimage view or any image that the user is required to supply to the other delegates that zoom the image in and out